aws/docker/worker/profile_plot_from_erddap.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import os
+import matplotlib
+matplotlib.use('AGG')
+import matplotlib.dates as mdate
+import boto3
+import cmocean
+import io
+import logging
+import matplotlib.cm as cm
+import matplotlib.pyplot as plt
+import netCDF4
+import numpy as np
+import traceback
+from datetime import datetime
+
+from erddapy import ERDDAP
+
+
+# In[ ]:
+
+
+dataset_id = 'ce_311-20200708T1723'
+
+#'ce_381-20171208T1841-delayed'
+#
+#'bios_minnie-20170616T1454'
+#'amelia-20180501T0000'
+#whoi_406-20160902T1700'
```

Comment:
Can you remove any extraneous comments, jupyter notebook artifacts, and debugging vars?

Reply:
done